### PR TITLE
Removal of dead code for rax2 -P

### PR DESCRIPTION
**Pull request by PaoloMonti42**
I found some unreachable code for rax2 -P.
The code was converting some python string/array into hex.
This functionality is already provided by rax2 -F.
Updated the manual to show it.

### CODE REVIEW

**Comment by radare** [comment]
Uh? Why kill it? Make it available and document it instead. Also travis is red

**Comment by PaoloMonti42** [comment]
It's just (not) doing a sub feature of -F, with zero added value. 
To make it available I would need to add some custom code at line 558, that was added to make -F work.
Also, no idea why travis would be red, it was green before I did a wrong commit and immediately reverted it.

**Comment by PaoloMonti42** [comment]
Magically not read anymore

File: binr/rax2/rax2.c [hunk 1]
```diff
@@ -107,7 +107,7 @@ static int help() {
 		"  -D      base64 decode        ;\n"
 		"  -E      base64 encode        ;\n"
 		"  -f      floating point       ;  rax2 -f 6.3+2.1\n"
-		"  -F      stdin slurp code hex ;  rax2 -F < shellcode.c\n"
+		"  -F      stdin slurp code hex ;  rax2 -F < shellcode.[c(py]\n"
```
Comment (radare) [inline binr/rax2/rax2.c]: lgtm

**Comment by radare** [comment]
thanks!

File: binr/rax2/rax2.c (hunk 1 above)
Comment (PaoloMonti42) [reply binr/rax2/rax2.c]: Darn, the typo. meant. to write [c/py] instead of [c(py]. Sorry

**Comment by radare** [comment]
Wanna take a look at this? https://github.com/radare/radare2/issues/2639

**Pull request merged**